packages/synapse-react-client/src/components/Markdown/index.ts

```diff
@@ -1,26 +0,0 @@
-export * from './widget'
-import type { MarkdownEditorProps } from './MarkdownEditor'
-import { MarkdownEditor } from './MarkdownEditor'
-import type { MarkdownSynapseProps } from './MarkdownSynapse'
-import Markdown from './MarkdownSynapse'
-import MarkdownSynapse from './MarkdownSynapse'
-import type { MarkdownWidgetProps } from './MarkdownWidget'
-import { default as MarkdownWidget } from './MarkdownWidget'
-import type { MarkdownCollapseProps } from './MarkdownCollapse'
-import { MarkdownCollapse } from './MarkdownCollapse'
-import type { MarkdownPopoverProps } from './MarkdownPopover'
-import { MarkdownPopover } from './MarkdownPopover'
-
-export {
```

Comment:
We now use `vite-plugin-dts` (which uses `@microsoft/api-extractor`) to create types, but it had a problem with this file. The problem went away when I deleted this file.

I wish I could explain what the problem was, but the error emitted by `@microsoft/api-extractor` literally just says

```
[vite:dts] Internal Error: Unable to analyze the export "MarkdownSynapse" in
/home/nickgrosenbacher/dev/Sage/synapse-web-monorepo/packages/synapse-react-client/dist/components/Markdown/index.d.ts

You have encountered a software defect. Please consider reporting the issue to the maintainers of this application.
<stack trace>
```

So I couldn't possibly get anywhere without trying to attach a debugger to the build task and trawling through that codebase.